### ISSUE TITLE
Reduce unnecessary queries for Flipper Controller route

### ISSUE THREAD

**Issue by bosawt** [department-of-veterans-affairs/vets-api#6165]
`GET FeatureToggles` is called often from the front-end, and when it's called, it makes a separate query for each Flipper entry. We should refactor this route so that we query a single time and then iterate over the resulting values. This happens here: `app/controllers/v0/feature_toggles_controller.rb`

**Comment by bosawt** [comment]
> Yes! When I wrote the endpoint, the idea was that the FE would make an API call with only the toggle names that it's interested in because it's possible that we could do more complex evaluations of data to decide who the feature is enabled for. But that hasn't happened (yet). Really, if the FE is going to make a request that calls all toggle values the response should be cached on the FE for a few minutes.

Ah yeah, that makes sense, I added this cause I was watching the Rails query logs and was wondering why this single route was just going absolutely crazy, ha. But does seem like a pretty easy initial solve at least to just cache on the frontend, as you suggest.

**Comment by bosawt** [comment]
Timing result of existing feature_toggle performance:
![Screen Shot 2021-07-06 at 4.51.50 PM.png](https://images.zenhubusercontent.com/5fb55f12a11eae1f7ca15bde/ee223274-78d6-437f-9492-0083bdc6e2d0)
Timing result of feature_toggle performance when only querying a single feature
![Screen Shot 2021-07-06 at 4.53.21 PM.png](https://images.zenhubusercontent.com/5fb55f12a11eae1f7ca15bde/f026786d-e15a-4874-bf62-db0cdcfc8ae2)

Basically it's a difference of roughly 275ms response turnaround to a 200ms response turnaround (so about 30% faster to only call a single feature). I think there must be a huge overhead for calls in general, 200ms for a single feature was only 30ms when I cURLed it from the terminal

**Comment by bosawt** [comment]
So given that, I think really there isn't much reason to do the work of splitting these out into separate feature calls. I think instead, we should just make it so the frontend calls it less. So basically I think the solution has to be to use localStorage. I'll try that now and see how it performs

**Comment by bosawt** [comment]
This is the timing for a fully mocked backend (no queries just returning a mocked features array: ![Screen Shot 2021-07-08 at 1.41.57 PM.png](https://images.zenhubusercontent.com/5fb55f12a11eae1f7ca15bde/0d572aee-7000-4c9e-97bc-6077a7c83920)
